packages/legend-extension-dsl-serializer/src/components/studio/BindingElementEditor.tsx

```diff
@@ -0,0 +1,450 @@
+      </div>
+    );
+  },
+);
+
+const BindingGeneralEditor = observer(
+  (props: { editorState: BindingEditorState; isReadOnly: boolean }) => {
+    const { editorState, isReadOnly } = props;
+    const editorStore = useEditorStore();
+    const binding = editorState.binding;
+    const schemaSets = Array.from(
+      editorStore.graphManagerState.graph.allOwnElements,
+    ).filter(
+      (type: PackageableElement): type is SchemaSet =>
+        type instanceof SchemaSet,
+    );
+    const schemaSetOptions = schemaSets.map((e) => ({
+      value: e,
+      label: e.path,
+    }));
+    const onSchemaSetChange = (
+      val: { label: string; value: SchemaSet } | null,
+    ): void => {
+      binding.setSchemaId(undefined);
+      return binding.setSchemaSet(val?.value);
+    };
+    const selectedSchemaSet = {
+      value: binding.schemaSet,
+      label: binding.schemaSet?.path,
+    };
+    const schemaIdOptions = selectedSchemaSet.value?.schemas.map((e) => ({
+      value: e.id,
+      label: e.id,
+    }));
+    const onSchemaIdChange = (
+      val: { label: string; value: string } | null,
+    ): void => binding.setSchemaId(val?.value);
+    const selectedSchemaId = {
+      value: binding.schemaId,
+      label: binding.schemaId,
+    };
+    const projectSelectorRef = useRef<SelectComponent>(null);
+    const contentTypeOptions = Object.values(CONTENT_TYPE).map((e) => ({
+      value: e,
+      label: e,
+    }));
+    const onContentTypeChange = (val: {
+      label: CONTENT_TYPE;
+      value: CONTENT_TYPE;
+    }): void => binding.setContentType(val.value);
+    const selectedContentType = {
+      value: binding.contentType,
+      label: binding.contentType,
+    };
+    return (
+      <div className="binding-general-editor">
+        <div className="binding-general-editor__section__header__prompt">
+          Select Content Type
+        </div>
+        <div>
+          <CustomSelectorInput
+            className="binding-general-editor__section__dropdown"
+            disabled={isReadOnly}
+            options={contentTypeOptions}
+            onChange={onContentTypeChange}
+            value={selectedContentType}
+            placeholder="Choose a content type"
+            darkMode={true}
+          />
+        </div>
+        <div className="binding-general-editor__section__header__prompt">
+          Select Schema Set
+        </div>
+        <div className="binding-general-editor__content">
+          <CustomSelectorInput
+            className="binding-general-editor__section__dropdown"
+            disabled={isReadOnly}
+            ref={projectSelectorRef}
+            options={schemaSetOptions}
+            onChange={onSchemaSetChange}
+            value={selectedSchemaSet}
+            isClearable={true}
+            placeholder="Choose a schema set"
+            darkMode={true}
+          />
+        </div>
+        <div className="binding-general-editor__section__header__prompt">
+          Select Schema Id
```

Comment:
-> `Schema ID` (this is the title of the block, not the description/placeholder)